Fixed modules: removed six dependency

JoyalMPaul opened the pull request:
This change resolves #1346 by removing outdated six.moves imports from r.in.srtm.region, r.in.nasadem, and r.in.usgs modules. Please let me know if I need to fix anything as this is my first contribution!

JoyalMPaul committed:
Fixed modules: removed six dependency

neteler committed:
Merge branch 'grass8' into remove-six-dependency

echoix reviewed (changes requested):
Can you also actually remove the installation (as a requirement) from https://github.com/OSGeo/grass-addons/blob/0df3f4a109ebb8a965e35e13bb943299cfc1352c/.github/workflows/requirements.txt#L3?

If you are able to try these modules with the change (making sure six is not available), it would be great. But keep in mind some other packages might install six as a dependency, so it might not be possible to really do that verification.

JoyalMPaul committed:
Removed unused module six from requirements.txt

JoyalMPaul committed:
Merge branch 'remove-six-dependency' of https://github.com/JoyalMPaul/grass-addons into remove-six-dependency

JoyalMPaul commented:
I removed six from requirements.txt since no other files used it, since everything else was using urllib. I also updated my branch since it was a few commits behind grass8. Hope that helps!

neteler committed:
Merge branch 'grass8' into remove-six-dependency
